view/frontend/web/js/view/payment/method-renderer/omise-cc-method.js

```diff
@@ -152,9 +161,7 @@ define(
                         self.getPlaceOrderDeferredObject()
                             .fail(
```

Comment:
`.fail(self.handlePaymentError)` is all you need here

Reply:
I was thinking about that.. and finally no... i don't like it.. as we are loosing function argument.
I prefere to see that `response`..